Models/UserModel.cs

```diff
@@ -9,6 +9,6 @@ public class UserModel
     {
         public string UserName { get; set; }
         public string Password { get; set; }
-        public string EmailAddress { get; set; }
+      public string EmailAddress { get; set; }
```

Comment:
You should keep your alignment and spaces consistent. To avoid changes like these.. You can f. ex use the shortcut Ctrl+K+D in Visual Studio to align your code.. (if you have the same setup as me :) )